frontend/src/pages/[userId]/index.tsx

```diff
@@ -34,21 +35,27 @@ export const CheckList: React.FC = () => {
 
   const [currentPhase, setCurrentPhase] = useState<number>(0);
   const [currentView, setCurrentView] = useState<number>(0);
+  const [isUserChecklist, setIsUserChecklist] = useState<boolean>(false);
 
   const { data: managedChecklists, refresh: refreshManagedChecklists } = useManagedChecklists();
   const {
     data: employeeChecklist,
     loaded,
-    loading,
     refresh: refreshChecklist,
   } = useChecklist((query?.userId as string) || username);
+  const { data: delegatedChecklists, refresh: refreshDelegatedChecklists } = useDelegatedChecklists();
 
   const managedChecklist = managedChecklists.filter((employee) => employee.employee.username === query?.userId)[0];
-  const data = currentView === 0 ? managedChecklist : employeeChecklist;
+  const delegatedChecklist = delegatedChecklists.filter((employee) => employee.employee.username === query?.userId)[0];
+  const data =
```

Comment:
Får felmeddelande när datat ska loopas.
`Cannot read properties of undefined (reading 'map')`